.github/workflows/rust.yml

```diff
@@ -43,102 +43,68 @@ jobs:
           isRust:
```

Comment:
As far as I'm concerned I still want the code to break early if it doesn't test properly on stable.